packages/customize-widgets/src/customize-control.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * WordPress dependencies
+ */
+import { render } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import SidebarBlockEditor from './components/sidebar-block-editor';
+import SidebarAdapter from './components/sidebar-block-editor/sidebar-adapter';
+
+const { wp } = window;
+
+const CustomizeControl = wp.customize.Control.extend( {
```

Comment:
Can we call this `SidebarControl` or something? `CustomizeControl` isn't very specific. Every control in the customizer is a customize control 🙂